### PR TITLE
Web accessibilty portfolio

### PR DESCRIPTION
**Pull request by lawrahkonwea**
In this portfolio, some changes were made which include;
- Changed page title from My-portfolio to Amaka's portfolio
-  Added alt attributes for the image tags that were missing
-  Changed headings title from h3 to h2 to achieve the correct flow of headings
-  Color contrast to make it more accessible
-  Resize - no issues found
-  Interactions -  add labels for the text inputs of the form
-  Moving content - no issues found
-  Multi-media - no issues found
-  The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,9 +68,9 @@ <h3>Tonic</h3>
           </div>
             <ul class="ul">
               <li><span class="canopy">CANOPY</span></li>
-              <img src="images/Counter (1).png" alt="">
+              <img src="images/Counter (1).png" alt="counter image">
```
Comment (anagudelogu) [inline index.html]: - [ ] There's something to take in mind here: 📜 If the image is not important for understanding the content — for example, it is just decoration or "eye candy" — it should have null alt (alt=""). One way to help determine if an image should have a null alt is to ask yourself: If the image was removed, would the user still get all the information from the page? Please, consider changing this across your project! 🤓 🛠️

File: index.html [hunk 2]
```diff
@@ -252,8 +252,11 @@ <h2>Contact Me</h2>
       needs coding. I’d love to help with it </p>
     </div>
     <form action="https://formspree.io/f/xgeqpkgk" method="post">
+      <label for="text" hidden="hidden">name:</label>
       <input type="text" name="text" id="text" placeholder="Enter name" maxlength="30" required>
+      <label for="email" hidden="hidden">email:</label>
       <input type="email" name="email" id="email" placeholder="Enter email" required>
+      <label for="formMessage" hidden="hidden">message:</label>
       <textarea name="formMessage" id="message" placeholder="Write your message here" maxlength="500"
```
Comment (anagudelogu) [inline index.html]: - [ ] Nice job adding labels! 💯 However, the approach you chose to hide them not only works visually but for assistive technologies such as screen readers. Please, check [this article](https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text) on how to hide labels properly, and use what is most suited to this use case! Consider changing this! 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
Hi Team! @lawrahkonwea @Wahaj-Ali, 

Great job making your portfolio more accessible! 🍾 🎆 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Good PR title and summary. 👍🏼 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @lawrahkonwea  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

-  Website deployed✔️ 



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**